types.go

```diff
@@ -105,7 +107,7 @@ type AuthorizationResponse struct {
 	CvvResultRaw         string
 	// Fields in case card details were updated through RTAU
 	RealTimeAccountUpdateStatus RTAUStatus
-	UpdatedExpiry               string
+	UpdatedExpiry               time.Time
 	UpdatedLast4                string
```

Comment:
Similar to what we do for avs and cvv responses, could we make a new type (`RTAUResponse`) which is a struct containing these fields and then add a field of that type to the AuthorizationResponse? Seems like a more natural way to group these related fields than just using a comment.

Reply:
oh yeah that makes a lot of sense, way cleaner